package.json

```diff
@@ -30,14 +30,6 @@
     "@babel/preset-react": "^7.14.5",
     "@babel/preset-typescript": "^7.14.5",
     "@babel/runtime": "^7.14.6",
-    "@emotion/react": "^11.7.1",
-    "@emotion/styled": "^11.6.0",
```

Comment:
I'm surprised pulling out emotion doesn't break Mui-material

https://www.npmjs.com/package/@mui/material#:~:text=npm%20install%20%40mui/material%20%40emotion/react%20%40emotion/styled

Reply:
no i haven't removed it completely, basically we had these two packages in dev dependencies as well as in dependencies so I just removed it from dev dependencies. these two packages are still present in the dependencies